.github/workflows/ci.yml

```diff
@@ -16,7 +16,10 @@ jobs:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [macos-latest, ubuntu-latest]
+        os:
+          - macos-13 # Intel
+          - macos-latest # Apple M1
```

Comment:
Based on GH official docs, the current `macos-latest` runner is based on Apple M1: 

<img width="758" alt="image" src="https://github.com/nix-community/fenix/assets/2049686/fffff713-4f62-40bb-bbb8-b1ff1c49a259">

**Source:**

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories